### ISSUE TITLE
The source of Covid-19 X-Ray datasets

### ISSUE THREAD

**Issue by scarlettouyang** [faniadev/covid19-xray-image-classification-using-CNN#1]
Hello! Thanks for your sharing! Could I ask that is your dataset open-sourced and where your dataset comes from? thanks a lot!

**Comment by faniadev** [comment]
yup, it's open-sourced! here's the link to the dataset source: https://github.com/ieee8023/covid-chestxray-dataset
